src/Umbraco.Web.UI.Client/src/views/common/infiniteeditors/insertfield/insertfield.html

```diff
@@ -52,11 +52,11 @@
                     <div class="control-group umb-control-group">
                         <div class="umb-el-wrap">
                             <div class="controls">
-                                <label class="control-label" >
+                                <label class="control-label">
                                     <localize key="templateEditor_recursive">Recursive</localize>
                                 </label>
+                                <umb-checkbox input-id="recursive" name="recursive" model="vm.recursive" disabled="!vm.field" />
```

Comment:
There is no need for the input-id and name attributes nor the label referencing it - It can be done like below where the label-key is also passed to the directive so it will be translated. Currently there will be 2 labels doing the same thing 😃

```
                                <umb-checkbox
                                    model="vm.recursive"
                                    text="Yes, make it recursive"
                                    label-key="templateEditor_recursiveDescr">
                                </umb-checkbox>
```

Reply:
@BatJan at the time of submitting this PR the `label-key` didn't exist.
Sometime you might want to use `input-id` to make other than the wrapping label toggle the checkbox.